app/controllers/questions_controller.rb

```diff
@@ -4,7 +4,12 @@ class QuestionsController < ApplicationController
   before_action :set_own_question, only: [:edit, :update, :destroy]
 
   def index
-    @questions = Question.all
+    if params[:sort] == "unanswered"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.